appveyor.yml

```diff
@@ -28,7 +28,7 @@ install:
   # replace with whatever version of ilastik triggered the appveyor
   - ps: cp -recurse C:\projects\ilastik $env:IlASTIK_ROOT\ilastik-meta\ilastik
   # Point to the current ilastik-meta
-  - python ilastik/scripts/devenv.py create -n %ENV_NAME% -c ilastik-forge conda-forge defaults -p ilastik-dependencies-no-solvers
+  - python ilastik/scripts/devenv.py --name %ENV_NAME% create
```

Comment:
aww I see. So this PR introduces a slightly different syntax from conda, but I guess we can live with that.

Reply:
I've moved `--name` into separate subcommands,

Reply:
awesome